docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,114 @@
+=============
+Accessibility
+=============
+
+All aspects of Django must be compatible with our accessibility standards,
+and with the specific support targets we aim for.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+Django aims for conformance with `WCAG 2.2`_ at the AA level.
+WCAG is the most well established standard for web accessibility. AA-level
+guidelines are the most common legal compliance target around the world.
+
+Wherever applicable, we also attempt to follow other best practices, such as:
+
+* WCAG 2.2 AAA-level guidelines.
+* The upcoming `WCAG 3.0`_ guidelines.
+* `ATAG 2.0`_ guidelines for authoring tools.
+
+.. note::
+    Do you want to learn more about accessibility without diving straight into
+    standards? We recommend `The A11Y Project`_,
+    a community-driven effort to make digital accessibility easier.
+
+.. _WCAG 2.2: https://www.w3.org/TR/WCAG22/
+.. _WCAG 3.0: https://www.w3.org/TR/wcag-3.0/
+.. _ATAG 2.0: https://www.w3.org/TR/ATAG20/
+.. _The A11Y Project: https://www.a11yproject.com/
+
+.. _accessibility-support-targets:
+
+Accessibility support targets
+=============================
+
+Beyond standards, we want to make sure Django actually works for people using
+a wide range of input devices and assistive technologies.
+The best way to do this effectively is to take accessibility considerations
+into account as part of designing features. If in doubt, consult
+with users who rely on assistive technologies or with accessibility experts.
```

Comment:
> in the spirit of a "shift-left" approach.

What happened to not using technical jargon in documentation, Thibaud? 🤔

Reply:
But also, all good points.

Reply:
😂 you got me

Reply:
I agree with the paragraph but as I mentioned in a PR-level comment, I think it's scary. I believe it should have links to concrete resources to quantify how many are enough "accessibility considerations" and who/where to reach out for help.

Reply:
Also noting [Andrew’s comment here](https://github.com/django/django/pull/17338#issuecomment-1843410088) as I believe this will be the right place to address this (TBC):

> As for the guidelines around strictness - I think it is especially important to say the implied part out loud (that progressive improvement in small parts is better than demanding perfection). We don't want to discourage people from contributing, we want to encourage them to instead make things a little bit better every time.

This was a reply to [my prev comment](https://github.com/django/django/pull/17338#issuecomment-1842250001)